### DISCUSSION TITLE
引擎跳转助手在google搜索界面会导致搜索框消失

### DISCUSSION THREAD

**Discussion by liuuuuu95** [F9y4ng/GreasyFork-Scripts#236] [问答专区(Question / Answer)]
目前使用的是cent最新版浏览器，内核Chromium 102.0.5005.167，在google搜索界面使用滚轮上下滑动时，会导致搜索框消失，同样的插件配置，在edge上就不会出现这个问题

**Comment by F9y4ng** [comment]
我在centBrowser 5.0.1002.354 64bit下不能复现你遇到的问题：

![test](https://github.com/F9y4ng/GreasyFork-Scripts/assets/15855068/b5493517-b446-4a2f-963c-8be693b4f7ff)

大概猜测以下几个原因：
* 梯子太慢，导致Google本身的JS没有加载完，滚动缩放的搜索栏就不会跟随滚动出现。（换个付费的高速梯子）
* 检查广告屏蔽扩展的某些订阅源是否有过度拦截。（关闭广告拦截扩展再试）
* 与其他脚本有冲突。（这需要你在本地一个一个去排除）

**Reply by liuuuuu95** [reply]
不知道呢，很抽象，我的pc上的cent就没问题。两个浏览器插件一样，目前就只有cent有这个问题，其他浏览器就没问题，视频左边是edge，右边是chrome，都是笔记本平台

https://github.com/F9y4ng/GreasyFork-Scripts/assets/109229395/b1377a4a-9a47-466c-a353-345132b9a104


[](url)

**Reply by F9y4ng** [reply]
如果其他相同内核浏览器可以正常执行脚本，说明脚本本身并没有异常。你需要在笔记本上检查centBrowser的设置及浏览器缓存。

从你的视频来看，很大概率是Google本身的JS脚本未正确运行。你可以先关闭脚本后，再测试滚动后搜索框是否正常，然后再逐一排查你的浏览器设置及本地环境问题。
